libs/blocks/merch/merch.js

```diff
@@ -226,7 +226,7 @@ export async function getUpgradeAction(
   imsSignedInPromise,
   [{ productArrangement: { productFamily: offerFamily } = {} }],
 ) {
-  if (options.entitlement === false) return undefined;
+  if (!options.upgrade) return undefined;
```

Comment:
don't we need both flags here?

Reply:
i was thinking about it, but then decided to remove 'entitlement' to have more flexibility.

- entitlement=true&upgrade=true: users will have both TWP modal and upgrade modal experience
- entitlement=true&upgrade=false: users will have only TWP modal 
- entitlement=false&upgrade=true: users will have only Upgrade modal, but no TWP
- entitlement=false&upgrade=false: only checkout page

Reply:
>entitlement=true&upgrade=true: users will have both TWP modal and upgrade modal experience
actually this is not correct, you need the **modal=true**